src/server/scripts/EasternKingdoms/Stratholme/stratholme.h

```diff
@@ -81,7 +82,10 @@ enum STRQuestIds
 
 enum STRSpellIds
 {
-    SPELL_BARON_ULTIMATUM               = 27861
+    SPELL_BARON_ULTIMATUM               = 27861,
+    SPELL_YSIDA_SAVED                   = 31912,
+    SPELL_YSIDA_CREDIT_EFFECT           = 31913,
+    SPELL_PERM_FEIGN_DEATH              = 29266
```

Comment:
cosmetic/codestyle:
`SPELL_PERM_FEIGN_DEATH (29266)` belongs on line 2 of this enum section, both by alphabetical sorting and by spell ID.